tasks/pages/index.tsx

```diff
@@ -66,9 +68,10 @@ const Dashboard: NextPage<PropsWithLanguage<DashboardTranslation, DashboardServe
             )
             )}
             right={width => (
-              <FeatureDetails
+              <NewsFeed
                 width={width}
-                features={featuresSchema.parse(jsonFeed)}
+                // TODO fix typing
+                localizedNews={localizedNewsSchema.parse(jsonFeed) as LocalizedNews}
```

Comment:
Why are you typechecking and parsing this here again?, `fetchLocalizedNews` already does this

Reply:
_See below_

@JannikWibker what would you suggest to get rid of this `as ...`?